cmd/common.go

```diff
@@ -422,6 +435,32 @@ func syncMain(ctx context.Context, filenames []string, dry bool, parallelism,
 	return nil
 }
 
+func validateSkipConsumersWithConsumerGroups(targetContent file.Content, config dump.Config) error {
+	if len(config.SelectorTags) == 0 {
+		printFn := color.New(color.FgYellow, color.Bold).PrintfFunc()
+		printFn("--skip-consumers-with-consumer-groups flag is added, but no select tags are specified.\n" +
+			"This can lead to unintended changes.\n\n")
+
+		ok, err := reconcilerUtils.Confirm("> Do you wish to continue? ")
```

Comment:
This would come up on diff command as well even though no writes to gateway are being made, correct?

Reply:
Yes, it would. We keep diff and sync similar in terms of output shown to the end-user.